include/micm/configure/solver_config.hpp

```diff
@@ -122,9 +128,35 @@ namespace micm
       std::filesystem::path species_config(config_dir / SPECIES_CONFIG);
       std::filesystem::path mechanism_config(config_dir / MECHANISM_CONFIG);
       std::filesystem::path reactions_config(config_dir / REACTIONS_CONFIG);
+      // Note tolerance_config is defined here but not used
       std::filesystem::path tolerance_config(config_dir / TOLERANCE_CONFIG);
 
-      // Current reaction configs should be either mechanism_config or reactions config
+      // Look for CAMP config file
+      std::filesystem::path camp_config(config_dir / CAMP_CONFIG);
+      if (std::filesystem::exists(camp_config))
+      {
+        std::cout << "Reading CAMP config " << camp_config << std::endl;
```

Comment:
Please remove or comment this out. Hopefully we will be adding a logger when we do #127, but for now I'd prefer we keep output relatively clear.